libtsuba/src/RDGCore.cpp

```diff
@@ -179,7 +188,7 @@ RDGCore::UpsertNodeProperties(
   auto written_prop_names = KATANA_CHECKED(UpsertProperties(
       props, &node_properties_, &part_header_.node_prop_info_list()));
   // store write properties into transaction context
-  txn_ctx->InsertNodePropertyWrite(std::move(written_prop_names));
+  txn_ctx->InsertNodePropertyWrite<std::unordered_set>(written_prop_names);
```

Comment:
Shouldn't we just take a range or pair of iterators? There is no reason for callers to know how a transaction context keeps track of access sets. They just need to do a multi-insert.

Reply:
Oh, the Container template idea came from me because Tyler used it and I liked it.  I prefer a single Container argument over begin/end iterators, but that is a personal thing.  I find `begin()/end()` unwieldy.